config_scheduler.go

```diff
@@ -14,6 +14,7 @@ type SchedulerConfig struct {
 	HostAllocator                 string  `bson:"host_allocator" json:"host_allocator" yaml:"host_allocator"`
 	HostAllocatorRoundingRule     string  `bson:"host_allocator_rounding_rule" json:"host_allocator_rounding_rule" mapstructure:"host_allocator_rounding_rule"`
 	HostAllocatorFeedbackRule     string  `bson:"host_allocator_feedback_rule" json:"host_allocator_feedback_rule" mapstructure:"host_allocator_feedback_rule"`
+	HostsOverallocatedRule        string  `bson:"hosts_overallocated_feedback_rule" json:"hosts_overallocated_feedback_rule" mapstructure:"hosts_overallocated_feedback_rule"`
```

Comment:
preference for keeping variable and bson/json consistent (maybe should just be hosts_overallocated_rule)

Reply:
yeah this was a regex goof on my part, thanks